controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -951,9 +952,23 @@ private void toSlime(Cursor object, Instance instance, DeploymentStatus status,
                     toSlime(instance.rotationStatus().of(instance.rotations().get(0).rotationId(), deployment),
                             deploymentObject);
                 }
-                if (!instance.rotations().isEmpty()) {
+                if ( ! recurseOverDeployments(request) && ! instance.rotations().isEmpty()) { // TODO jonmv: clean up when clients have converged.
                     toSlime(instance.rotations(), instance.rotationStatus(), deployment, deploymentObject);
                 }
+
+                JobType.from(controller.system(), deployment.zone())
```

Comment:
Just noticed I wrote the wrong path on slack when requesting this feature. Could you please move it to deployment? Where `endpointStatus` is.

Reply:
It would bring me the greatest pleasure to do so.

Reply:
✅